### PR TITLE
Develop links

### PR DESCRIPTION
**Pull request by csalvador**

### CODE REVIEW

File: htdocs/core/lib/files.lib.php [hunk 1]
```diff
@@ -1044,7 +1045,24 @@ function dol_add_file_process($upload_dir,$allowoverwrite=0,$donotupdatesession=
...
+			$linkObject->label = GETPOST('label', 'alpha');
+			$res = $linkObject->create($user);
+			$langs->load('link');
+			if ($res > 0) {
+				setEventMessage($langs->trans("LinkComplete"));
+			} else {
+				setEventMessage($langs->trans("ErrorFileNotLinked"), 'errors');
+			}
+		}
+    }
```
Comment (rdoursenaud) [inline htdocs/core/lib/files.lib.php]: indent less

File: htdocs/core/tpl/doc2.tpl.php [hunk 2]
```diff
@@ -0,0 +1,45 @@
+<?php
```
Comment (rdoursenaud) [inline htdocs/core/tpl/doc2.tpl.php]: maybe use a better name ?

Also don't forget the copyright headers !

File: htdocs/core/tpl/document_actions.tpl.php [hunk 3]
```diff
@@ -0,0 +1,70 @@
+<?php
```
Comment (rdoursenaud) [inline htdocs/core/tpl/document_actions.tpl.php]: Don't forget the copyright header

File: htdocs/adherents/document.php [hunk 4]
```diff
@@ -60,47 +61,31 @@
...
 
-$form = new Form($db);
-$object=new Adherent($db);
-$membert=new AdherentType($db);
 
 llxHeader();
 
 if ($id > 0)
 {
-    $result=$object->fetch($id);
```
Comment (rdoursenaud) [inline htdocs/adherents/document.php:103]: Looks like a bug, no ?

File: htdocs/projet/tasks/document.php [hunk 5]
```diff
@@ -244,15 +229,18 @@
 
 	print '<br>';
 
-
-	// Affiche formulaire upload
+    $modulepart = 'projet';
```
Comment (rdoursenaud) [inline htdocs/projet/tasks/document.php]: Strange indent in GitHub

File: htdocs/adherents/document.php (hunk 4 above)
Comment (csalvador) [inline htdocs/adherents/document.php:103]: Nope :)

**Pull request merged**